internal/services/appconfiguration/app_configuration_feature_resource.go

```diff
@@ -200,20 +200,25 @@ func (k FeatureResource) Create() sdk.ResourceFunc {
 			}
 			featureKey := fmt.Sprintf("%s/%s", FeatureKeyPrefix, rawKey)
 
+			nestedItemId, err := parse.NewNestedItemID(client.Endpoint, featureKey, model.Label)
+			if err != nil {
+				return err
+			}
+
+			deadline, ok := ctx.Deadline()
+			if !ok {
+				return fmt.Errorf("could not determine context deadline for create for %s", nestedItemId)
```

Comment:
Users can't do anything about this since it's an internal error. For internal errors we typically specify:
```suggestion
				return fmt.Errorf("internal-error: context had no deadline")
```